README.md

```diff
@@ -24,11 +24,10 @@ Core features include:
 
 <!-- toc -->
 
-- [Talawa API](#talawa-api)
```

Comment:
Why have you done this change ? I think it's not needed.

Reply:
I see, this is getting committed because of the `npm run update:toc` command in the pre-commit.